app/core/config.py

```
@@ -0,0 +1,293 @@
+        
+        Raises:
+            ValueError: If the URL does not start with 'postgresql://', 'postgres://', or 'sqlite:///'.
+        
+        Example:
+            >>> DatabaseConfig.validate_database_url("postgresql://user:pass@localhost/db")
+            'postgresql://user:pass@localhost/db'
+        
+        Generated by CodeRabbit
+        """
+        if not v.startswith(("postgresql://", "postgres://", "sqlite:///")):
+            raise ValueError("Invalid database URL format")
```

Ruff (0.11.9)

Avoid specifying long messages outside the exception class

(TRY003)